lib/analysis_options.2.0.0.yaml

```
@@ -52,29 +52,33 @@ dart_code_metrics:
     - no-equal-then-else
     - prefer-trailing-comma
     - member-ordering:
-        alphabetize: false # TODO set true after code-dart_code_metrics update
+        alphabetize: false
         order:
           - public-static-const-fields
-          - private-static-const-fields
           - public-static-final-fields
-          - private-static-final-fields
           - public-static-fields
-          - private-static-fields
+          - public-static-methods
+          - constructors
+          - named-constructors
+          - factory-constructors
+          - overridden-final-fields
           - public-final-fields
           - private-final-fields
+          - overridden-fields
           - public-fields
-          - public-getters-setters
           - private-fields
+          - overridden-getters-setters
+          - public-getters-setters
           - private-getters-setters
-          - constructors
-          - named-constructors
-          - factory-constructors
           - overridden-methods
-          - public-static-methods
           - public-methods
           - protected-methods
-          - private-static-methods
+          - dispose-method
           - private-methods
+          - private-static-const-fields
+          - private-static-final-fields
+          - private-static-fields
+          - private-static-methods
 
```

Some experiments later... This look like what I was trying to achieve.
- public static members at the top
- private static members at the very bottom
- getters and setters could be everywhere 🪄 

```suggestion
          - public-static-fields
          - public-static-methods
          - constructors
          - named-constructors
          - factory-constructors
          - public-fields
          - private-fields
          - overridden-methods
          - public-methods
          - protected-methods
          - dispose-method
          - private-methods
          - private-static-fields
          - private-static-methods
```